.studio/common

```diff
@@ -231,6 +231,7 @@ function build_all() {
     [[ -f "$d/habitat/plan.sh" ]] || continue # skip what can't be built
     if [[ "$d" = "/src/components/automate-chef-io" ]] && [[ -n "$AUTOMATE_OSS_BUILD" ]]; then
         log_line "Skipping build of $d since AUTOMATE_OSS_BUILD is set."
+        continue
```

Comment:
Shouldn’t this be a `return` instead?

Reply:
I don't think so because this is inside a for loop where we are trying to build all of the packages and the point of the variable is "build all packages, except those that aren't open source"

Reply:
You are right. Disregard